sycl/doc/extensions/ITTAnnotations/ITTAnnotations.rst

```diff
@@ -77,3 +81,7 @@ For this purpose we reserve a 1-byte specialization constant numbered
 ``4285822057`` (``0xFF747469``). The users/tools/runtimes should set this
 specialization constant to non-zero value to enable the ITT annotations
 in SYCL device code.
+
+The specialization constant value is controlled by
+INTEL_ENABLE_OFFLOAD_ANNOTATIONS environment variable. Tools, that support ITT
```

Comment:
Should this environment variable be added to [EnvironmentVariables.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md)?  Note that all those environment variables start with the "SYCL_" prefix.  Should this one?

Reply:
>> Should this environment variable be added to EnvironmentVariables.md? 

Not sure how it can be used outside of the tools that know about this var. But, since we check it, I agree that we should mention it in the EnvironmentVariables.md.

>> Note that all those environment variables start with the "SYCL_" prefix. Should this one?

This variable is used not only for the SYCL RT, that's why it has no SYCL_ prefix.